res/mac/FreenetTray/src/LICENSE

```diff
@@ -1,81 +1,45 @@
 Original code:
 
 This program is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
+it under the terms of the GNU General Public License version 2, 
+as published by the Free Software Foundation.
```

Comment:
I don't think you can just tighten this part of the license (i.e. dropping the *or any later version* clause): other contributors to this tray app have licensed their work under GPLv2 *or later*, both GPLv2 and GPLv3 require to redistribute with license/copyright info intact.
Or should I consider this a full rewrite, where no original work is inherited? In that case I'm fine with this change.